server/server.go

```diff
@@ -229,6 +230,13 @@ func NewServer(cfg *config.Config, driver IDriver) (*Server, error) {
 	return s, nil
 }
 
+func (s *Server) initDefaults() {
```

Comment:
It is better to add all https://github.com/pingcap/tidb/blob/master/tidb-server/main.go#L451 in this method.